static/js/containers/DashboardPage.js

```diff
@@ -649,6 +664,15 @@ class DashboardPage extends React.Component {
     return null;
   };
 
+  renderLearnersInProgramCard(programID: number) {
+    const { programLearners } = this.props;
+    let learnersInProgramCard;
+    if(R.pathOr(false, [programID, 'data', 'learners'],programLearners)) {
```

Comment:
should we still display the card if the result of the `pathOr` call is `[]`? If you do `if ([]) { foo() }` then `foo` will be executed.